WifiDirectTest/app/src/androidTest/java/test/microsoft/com/wifidirecttest/ApplicationTest.java

```diff
@@ -0,0 +1,13 @@
+package test.microsoft.com.wifidirecttest;
+
+import android.app.Application;
+import android.test.ApplicationTestCase;
+
+/**
+ * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
+ */
+public class ApplicationTest extends ApplicationTestCase<Application> {
+    public ApplicationTest() {
+        super(Application.class);
+    }
+}
```

Comment:
Ideally we need the tests to be automated so that someone who wants to re-run these experiments themselves can do so quickly and easily. For example, I could see a test that checks that the onPause() event files (so we know we are in the background) and then waits until it detects someone else before declaring itself as passing. Etc.